src/main/java/communication/Subscriber.java

```diff
@@ -68,6 +68,11 @@ public Subscriber(String address, int port, String secret, EncryptionAlgorithm a
 	protected void interpreteReceivedEnvelope(Envelope envelope, ZMQ.Socket responseSocket) {
 		Message m = new Message();
 		try {
+			// start task that checks on missed messages if a messageID was set for the message
+			if (envelope.getMessage().getMessageID() != null) {
+				fBase.messageIDEvaluator.addReceivedMessageID(envelope.getMessage().getMessageID());
```

Comment:
change field name - sounds too much like IDE

Reply:
Do you have a suggestion, we would have to also change the name of the MessageID class.

Reply:
maybe: messageIdEvaluator